deployment/secure_api_server.py

```
@@ -933,10 +935,10 @@ def add_to_blacklist():
         
         ip = data['ip']
         rate_limiter.add_to_blacklist(ip)
-        logger.info(f"Added {ip} to blacklist")
+        logger.info("Added %s to blacklist", ip)
         return jsonify({'message': f'Added {ip} to blacklist'})
     except Exception as e:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)